simphony/io/h5_cuds.py

```diff
@@ -194,6 +238,28 @@ def get_mesh(self, name):
             raise ValueError(
                 'Mesh \'{n}\` does not exist'.format(n=name))
 
+    def get_lattice(self, name):
+        """Get lattice from file.
+
+        The returned lattice can be used to query
+        and change the related data stored in the file. If the
+        file has been closed then the lattice should
+        no longer be used.
+
+        Parameters
+        ----------
+        name : str
+            name of lattice to return
+        """
+        if name in self._file.root.lattice:
+            F = self._file.root.lattice._v_leaves[name].attrs
+            lat = FileLattice(self, name, F.type, F.base_vect,
```

Comment:
Changed to self._file. FileLattice constructor needs a reference to the file it uses.